server/data/contacts.ts

```diff
@@ -171,13 +171,13 @@ export const getContactById = async (userId: string, applicationId: string, cont
  */
  export const updateContact = async (userId: string, applicationId: string, contactId: string, name: string, phone: string, email: string): Promise<ContactObject> => {
 
-    if(!checkObjectId(userId))
+    if(!userId || !checkObjectId(userId))
         throw new Error('Invalid userId');
 
-    if(!checkObjectId(applicationId))
+    if(!applicationId || !checkObjectId(applicationId))
         throw new Error('Invalid applicationId');
 
-    if(!checkObjectId(contactId))
+    if(!contactId || !checkObjectId(contactId))
         throw new Error('Invalid contactId');
 
```

Comment:
Why do you not check !name, !phone, !email ?

Reply:
Below this line there are already if statements that check for those fields